### ISSUE TITLE
Redirect tablet-related error notifications to OpenTabletDriver's website

### ISSUE THREAD

**Issue by CherryZeff** [ppy/osu#20324]
### Type

Game behaviour

### Bug description

When starting the game, it displays a message saying "device:failed to set device feature: 02-02"

### Screenshots or videos

![1](https://user-images.githubusercontent.com/113616487/190432061-ffb532f5-737b-4fc3-8011-fea5892a4b46.png)
![2](https://user-images.githubusercontent.com/113616487/190432065-7784267d-3aa3-48b8-8529-503450bd36fd.png)

### Version

2022.911.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9576178/database.log)
[network.log](https://github.com/ppy/osu/files/9576180/network.log)
[performance.log](https://github.com/ppy/osu/files/9576181/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9576182/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9576183/updater.log)

**Comment by frenzibyte** [comment]
"Failed to set device feature" is an OpenTabletDriver-side issue. Will keep this open to track redirecting the notifiaction to there.

**Comment by frenzibyte** [comment]
I've spent some time coming up with something better for this just so we don't allow raw `OpenTabletDriver` logs entering user notifications, and I've came up with the following:

<img width="469" alt="CleanShot 2022-09-16 at 00 42 36@2x" src="https://user-images.githubusercontent.com/22781491/190514208-1ff4aa1f-54a9-4ed8-befd-b5d65f031a34.png">

(with the last notification directing [here](http://opentabletdriver.net/Tablets) on click).

Icon is very not final, but it's the closest here. @peppy would like to hear your thoughts on this before I consider opening PRs for this.
